### PR TITLE
Move some scripts we don't use anymore out of the images

### PR DESCRIPTION
**Pull request by JayH5**

### CODE REVIEW

**Comment by JayH5** [comment]
I can think of maybe one place where `eval-args.sh` is used and it doesn't really need to be. Most things have an entrypoint script instead.

File: README.md [hunk 1]
```diff
@@ -54,24 +54,16 @@ There is a subtle difference between the two forms of the [Dockerfile `CMD` dire
 Another problem is that if the command is not wrapped in a shell, variables aren't evaluated in the `CMD` instruction because there is no shell to ever resolve the variables' values.
 
 ##### Our solution:
-* A Bash script that evaluates each part of a command to resolve all the variables' values and then `exec`s the resulting command.
 * The images have this script launching `dumb-init` as the default `ENTRYPOINT`.
```
Comment (jerith) [inline README.md]: Which script?

Comment (JayH5) [inline README.md]: er.. eval-args.sh. So this is out of date.. good catch.

**Comment by jerith** [comment]
:+1:

**Pull request merged**